examples/data-objects/focus-tracker/src/FocusTracker.ts

```diff
@@ -33,6 +34,15 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
      */
     private readonly focusMap = new Map<string, Map<string, boolean>>();
 
+    /**
+     * Local map of mouse position status for clients
+     *
+     * ```
+     * Map<userId, Map<clientid, hasFocus>>
+     * ```
+     */
+    private readonly posMap = new Map<string, Map<string, [number, number]>>();
```

Comment:
Consider defining a type to help explain the map, maybe also using an object rather than tuple with named properties.

Reply:
I'm unsure what you mean by defining a type to explain the map. Do you mean a type like having a `MousePosition` type instead of ` [number, number] `?

Reply:
Yeah, something like that -- the `MousePosition` type could also be something like
```ts
interface MousePosition {
    x: number;
    y: number;
}
```
Rather than a tuple, which could make the code a bit more self-documenting in the consumption (e.g. `position.x` rather than `position[0]`)